src/Card/Card.tsx

```diff
@@ -0,0 +1,100 @@
+/** @jsx jsx */
+import * as colors from "../colors";
+import { base } from "../typography";
+import { jsx } from "@emotion/core";
+import React from "react";
+import PropTypes from "prop-types";
+
+interface Props {
+  children?: React.ReactNode;
+  title?: React.ReactNode;
+  className?: string;
+  description?: React.ReactNode;
+  titleChildren?: React.ReactNode;
+  forceNoChildPadding?: boolean;
+  size?: "standard" | "large";
+}
+
+export const Card: React.FunctionComponent<Props> = ({
+  children,
+  forceNoChildPadding,
+  title,
+  titleChildren,
+  description,
+  size,
+  ...otherProps
+}) => (
+  <div
+    {...otherProps}
+    css={{
+      backgroundColor: colors.white,
+      color: colors.black.base,
+      boxShadow: `0 4px 8px 0 rgba(0, 0, 0, .04)`,
+      borderRadius: `0.5rem`,
+      borderWidth: 1,
+      borderColor: colors.silver.dark,
+      paddingLeft: 24,
+      paddingRight: 24,
+      margin: "2rem",
+      paddingTop: size === "large" ? 16 : 28,
+      paddingBottom: size === "large" ? 16 : 28,
+    }}
+  >
+    <div css={{ display: "flex" }}>
+      <div css={{ flex: "1 1 0%" }}>
+        <div css={{ display: "block" }}>
+          {title && (
+            <div
+              css={{
+                display: "flex",
+                color: colors.black.base,
+                ...(size === "large" ? base.xxlarge : base.large),
+              }}
+            >
+              <span
+                css={{
+                  fontWeight: 600,
+                  flex: "1 1 0%",
+                  lineHeight: 1.5,
+                  overflow: "hidden",
+                  textOverflow: "ellipsis",
+                  whiteSpace: "nowrap",
+                  paddingRight: "1rem",
+                }}
+              >
+                {title}
+              </span>
+            </div>
+          )}
+          {description && (
+            <div
+              css={{
+                ...base.base,
+                color: colors.grey.base,
+                maxWidth: titleChildren ? "42rem" : "",
+                paddingBottom:
+                  !!children && !forceNoChildPadding ? "1.5rem" : "",
+              }}
+            >
+              {description}
+            </div>
+          )}
+        </div>
+      </div>
+      {titleChildren && (
+        <div css={{ flex: "none", marginLeft: "1rem" }}>{titleChildren}</div>
```

Comment:
Do we want to automatically align this to the right? We can do that by making the _other_ child in the flexbox have a `margin-right` of `auto`.